### ISSUE TITLE
Wifi module fail to load after initial setup

### ISSUE THREAD

**Issue by auranja** [arch-linux-gui/arch-linux-gui-plasma#2]
Hello.
I tried the kde themed version, everything worked fine during the install and Arch booted smoothly. Good job for you.
However, I've find that my Wi-Fi doesn't work after install. I'm using an Intel Wi-Fi 6 AX200 Adapter integrated in my laptop's motherboard. Right after the first boot, the networks tab in the plasma taskbar is blank, and there are no Wi-Fi devices shown when using `ip link`.
What is weird is, when I use `lspci -k`, lspci can detect the existence of my Wi-Fi card and it stated there are two kernel modules `iwlwifi, wl`  installed and iwlwifi driver in use. 
I tried `modprobe wl` and `modprobe iwlwifi`. Interestingly after serveral times of reboot ( I was desperatingly figuring out what happened ), the Wi-Fi worked without any sign or notice.
I reinstalled from your ISO several times and tried both offline and online installation. The Wi-Fi worked well in the live environment and the installer did passed the credentials to the new install, and the settings of connection can be seen in kde settings. When I click on the saved connection, kde also pop out a notification telling me that kde failed to get secrets. I don't know if this is related to the problem.
Since the Wi-Fi recovered weirdly, I didn't save any logs. However you can ask me ahead if you need more details. Thank you.

**Comment by kiocosta** [comment]
Unfortunately, nothing seems to work properly on this project. Tried installing it twice.
After the initial update, it starts using two competing Network Manager which makes ethernet unavailable.
Systemd is unable to stop one of the services, locale isn't set properly and the system won't reboot correctly due to problems in swap file apparently.

All these problems occur just after a fresh install. Huge thumbs down to this project, unfortunately it was a waste of time for me.

**Comment by auranja** [comment]
You are right, I figured out there was two network manager conflicting and I have to disable one to make Internet work.

**Comment by yehonatan2020** [comment]
Try removing iwd and then enaling wpa_supplicant
paru -R iwd 
sudo systemctl enable wpa_supplicant
systemctl reboot
